src/scripts/main.js

```diff
@@ -1 +1,54 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => resolve());
+
+  setTimeout(() => reject(new Error('Promise was rejected!')), 3000);
+});
+
+const secondPromise = new Promise((resolve) => {
+  document.addEventListener('click', () => resolve());
+  document.addEventListener('contextmenu', () => resolve());
+});
+
+const thirdPromise = new Promise((resolve) => {
+  let leftClick = false;
+  let rightClick = false;
+
+  document.addEventListener('click', () => {
+    leftClick = true;
+
+    if (rightClick) {
+      resolve();
+    }
+  });
+
+  document.addEventListener('contextmenu', () => {
+    rightClick = true;
+
+    if (leftClick) {
+      resolve();
+    }
+  });
+});
+
+const showMessage = (count, success = true) => {
```

Comment:
Instead of boolean value accept type of the notification
```suggestion
const showMessage = (count, type) => {
```